### ISSUE TITLE
Execption using

### ISSUE THREAD

**Issue by f3rnandomoreno** [new3Rs/current-visitors#1]
Hi!
I have tried to use the package but when currentVisitors is called I have got an exception on server side:
`Exception while invoking method 'current-visitors' MongoError: '$unset' is empty. You must specify a field like so: {$mod: {<field>: ...}}`

**Comment by f3rnandomoreno** [comment]
It says that $set has not be empty. And it is empty now.

``` javascript
updateVisitors = function() {
    var id, path, set, unset, _ref;
    set = {};
    unset = (_ref = CurrentVisitors.findOne()) != null ? _ref : {};
    delete unset._id;
    for (id in connections) {
      path = connections[id];
      path = path.replace(/\./g, '%2E');
      if (set[path] != null) {
        set[path] += 1;
      } else {
        set[path] = 1;
      }
      delete unset[path];
    }
    CurrentVisitors.upsert({}, {
      $set: set,
      $unset: unset
    });
  };
```

**Comment by f3rnandomoreno** [comment]
Here the solution:

``` javascript
  updateVisitors = function() {
    var id, path, set, unset, _ref;
    set = {};
    unset = (_ref = CurrentVisitors.findOne()) != null ? _ref : {};
    delete unset._id;
    for (id in connections) {
      path = connections[id];
      path = path.replace(/\./g, '%2E');
      if (set[path] != null) {
        set[path] += 1;
      } else {
        set[path] = 1;
      }
      delete unset[path];
    }

    if (_.isEmpty(set)) {

      if (_.isEmpty(unset)) {
        //do nothing
        return;
      }

      CurrentVisitors.upsert({}, {
        $unset: unset
      });

      // it is updated yet
      return;
    }

    if (_.isEmpty(unset)) {
      if (_.isEmpty(set)) {
        //do nothing
        return;
      }

      CurrentVisitors.upsert({}, {
        $set: set
      });

      // it is updated yet
      return;
    }
    CurrentVisitors.upsert({}, {
      $set: set,
      $unset: unset
    });
  };
```

**Comment by y-ich** [comment]
Thank you for your suggestion.
I modified it by bad8c8552d02419ecd105cbd280d485818962475
fixed #1

**Comment by f3rnandomoreno** [comment]
I like your solution :)
